hadoop/README.md

```diff
@@ -46,16 +46,16 @@ Start the JDG Server
 Start Apache Flink
 --------------------
 
-1. Download and unpack v1.0 of [Apache Flink](https://flink.apache.org/downloads.html#binaries). On Windows, you can use 7-zip to extract the .tgz file.  
+1. Download and unpack v1.7 of [Apache Flink](https://flink.apache.org/downloads.html#binaries). On Windows, you can use 7-zip to extract the .tgz file.  
```

Comment:
Could you change this line to:
```
1. Download and unpack v1.7 of [Apache Flink with Hadoop](https://www-eu.apache.org/dist/flink/flink-1.7.1/flink-1.7.1-bin-hadoop28-scala_2.11.tgz). On Windows, you can use 7-zip to extract the .tgz file.  
```

Downloading the "Flink only server" will cause the job to fail